rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -168,8 +168,8 @@ disabling stored fields removes fetch sub phases:
 ---
 dfs knn vector profiling:
   - skip:
-      version: ' - 8.6.99'
-      reason: multi-knn dfs profiling implemented in 8.7.0
+      version: ' - 8.11.99'
```

Comment:
Are we sure we want to skip older versions here over creating a new test for 8.12+?

Reply:
@kderusso, @benwtrent 

Should we create a seperate test section (vector_ops_count) for 8.12+?

Reply:
Yes we should. We really don't want to lose coverage and its very possible for folks to do `profile: true` in an environment where nodes have different versions. We need to ensure our APIs still work

Reply:
Updated.